### ISSUE TITLE
[BUG] Topic does not have schema to check when the function attach the topic that kop created.

### ISSUE THREAD

**Issue by codelipenghui** [streamnative/kop#178]
**Describe the bug**
Topic does not have schema to check when the function attach the topic that kop created.

**To Reproduce**
1. publish some messages to kop topic
2. create a function and uses the above topic
3. see error "Topic does not have schema to check"

**Expected behavior**
Pulsar function can consume messages from the topic that kop created.

**Comment by sijie** [comment]
I don't think we need to fix the issue here. People can create a function with a serde. So the function itself will not check the schema.
